components/org.wso2.carbon.identity.application.authenticator.hypr.common/src/main/java/org/wso2/carbon/identity/application/authenticator/hypr/common/model/DeviceAuthenticationRequest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.authenticator.hypr.common.model;
+
+import org.wso2.carbon.identity.application.authenticator.hypr.common.constants.HyprAuthenticatorConstants;
+import org.wso2.carbon.identity.application.authenticator.hypr.common.web.HYPRWebUtils;
+
+import java.security.NoSuchAlgorithmException;
+import java.util.Map;
+
+/**
+ * Model class for the request which initiates the device authentication.
+ */
+public class DeviceAuthenticationRequest {
+
+    private final String namedUser;
+    private final String machine = HyprAuthenticatorConstants.HYPR.MACHINE_VALUE;
```

Comment:
Why do we need to assign this to machine variable instead of using the direct value

Reply:
This was actually suggested during the code review session to access the value through the constants.

Reply:
can't we directly use `HyprAuthenticatorConstants.HYPR.MACHINE_VALUE`

Reply:
In order to do json mapping the variable have to be defined globally. Hence it is not possible to directly use the variable as you suggested.